### PR TITLE
[AI Deploy Beta] - capabilities and limitations update

### PR DESCRIPTION
**Pull request by eleapttn**
Doc improvement since AI Deploy Beta:

- add new features (scaling strategies, rolling upgrade)
- add limitation (docker-compose)

### CODE REVIEW

**Comment by eleapttn** [comment]
Just waiting for the review of AI Solutions team. Please wait before merging ⏳

**Review by baaastijn — APPROVED** [review]
Many thanks, good for me !

**Comment by OVH-Team-Guides** [comment]
Thank you for your contribution @eleapttn

**Pull request merged**
